packages/flutter_tools/lib/src/macos/build_macos.dart

```diff
@@ -85,8 +85,6 @@ Future<void> buildMacOS({
       'OBJROOT=${globals.fs.path.join(flutterBuildDir.absolute.path, 'Build', 'Intermediates.noindex')}',
       'SYMROOT=${globals.fs.path.join(flutterBuildDir.absolute.path, 'Build', 'Products')}',
       'COMPILER_INDEX_STORE_ENABLE=NO',
-      if (buildInfo.treeShakeIcons)
```

Comment:
@dnfield I don't think this is needed since its otherwise in the generated xcconfig

Reply:
That sounds right - did you test it locally?

Building flutter gallery with this option should result in a smaller MaterialIcons-Regular.ttf file (something like 13 kb instead of 100+kb).

Reply:
Now it actually works - the flag name was out of date in the invocations